$MirOS: ports/www/lynx/patches/patch-lynx_help_lynx_help_main_html,v 1.2 2007/05/20 12:06:19 tg Exp $

	we install these, so link to them

--- lynx_help/lynx_help_main.html.orig	Thu May 17 22:52:59 2007
+++ lynx_help/lynx_help_main.html	Sun Mar  9 17:50:39 2008
@@ -23,6 +23,10 @@
  -- credits, copyright etc
 <li><a href="lynx-dev.html">About Lynx-Dev</a>
  -- the developers &amp; how to contact them
+<li><a href="../lynx_doc/samples/">Lynx samples</a>
+ -- the samples contained in the source distribution
+<li><a href="../lynx_doc/test/">Lynx tests</a>
+ -- the test cases contained in the source distribution
 </ul>
 
 <h2>Other sources of Lynx help:</h2>
